openid-connect-server-spring-boot/src/main/java/org/mitre/TaskConfig.java

```diff
@@ -0,0 +1,46 @@
+package org.mitre;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+import org.mitre.oauth2.service.impl.DefaultOAuth2ProviderTokenService;
+import org.mitre.openid.connect.service.ApprovedSiteService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.scheduling.annotation.SchedulingConfigurer;
+import org.springframework.scheduling.config.ScheduledTaskRegistrar;
+
+@Configuration
+@EnableScheduling
+public class TaskConfig implements SchedulingConfigurer {
+	@Autowired
+    private DefaultOAuth2ProviderTokenService defaultOAuth2ProviderTokenService;
+	
+	@Autowired
+    private ApprovedSiteService approvedSiteService;
+	
+	@Bean(destroyMethod="shutdown")
+	public Executor taskScheduler() {
+		return Executors.newScheduledThreadPool(5); // TODO make configureable
+	}
+	
+	// TODO what's exaclty the difference between, taskScheduler and taskExecutor?
+	
+	@Scheduled(fixedDelay = 30000, initialDelay = 60000)
```

Comment:
Shouldn't this use something like `fixedDelayString = "${openid.connect.server.scheduler.token.fixedDelay}"` same for initial